package.json

```diff
@@ -33,5 +33,9 @@
   "repository": {
     "type": "git",
     "url": "git@github.com:gengo/style-guide.git"
+  },
+  "scripts": {
+    "dev": "node_modules/grunt-cli/bin/grunt dev",
```

Comment:
You don't need to prefix by the path. All the bins in `node_modules` are automatically added to `scritpts` path.
So: `"dev": "grunt dev"`